pftools/python/parflow/tools/core.py

```diff
@@ -223,6 +223,37 @@ def __init__(self, name, basescript=None):
         else:
             settings.set_working_directory()
 
+    @classmethod
+    def from_definition(cls, file_path):
+        """Method to generate a Run object from a file.
+
+        Currently accepts the following input file types:
+            yaml
+            pfidb
+
+        Args:
+            file_path (str): path to the file to read in
+
+        Returns:
+            A new Run object
+        """
+        file_path = Path(file_path)
```

Comment:
We need to support env variable like we do in our other path

Reply:
Do you think we should use `get_absolute_path()` so that it both expands the variables, and uses the `settings.WORKING_DIRECTORY` if the path is not absolute?

Reply:
yes

Reply:
Although the `pf_work_dir` could be confusing in that case since it is not set. It might be better to use env_replace + current python chdir.

Reply:
This is resolved by https://github.com/parflow/parflow/pull/289/commits/05d15de1cf7fb71a5b765714e8058bdbe282c36c